governance/eval-gate.py

```
@@ -0,0 +1,199 @@
+class Red(Exception):
+    pass
+
+
+def load_report(path: str) -> dict:
+    try:
+        with open(path, encoding="utf-8") as f:
+            data = json.load(f)
+    except (OSError, json.JSONDecodeError) as e:
+        die2(f"报告不可读: {path}: {e}")
+    if not isinstance(data, dict) or not isinstance(data.get("metrics"), dict):
+        die2(f"报告结构非法（须含 metrics 对象）: {path}")
```

Ruff (0.16.2)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)